yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -353,7 +353,7 @@ private[yarn] class YarnAllocator(
 
     } else if (missing < 0) {
       val numToCancel = math.min(numPendingAllocate, -missing)
-      logInfo(s"Canceling requests for $numToCancel executor containers")
+      logInfo(s"Canceled requests for $numToCancel executor container(s)")
```

Comment:
Maybe an other solution would be to change the `Driver requested a total number of $requestedTotal executor(s).` to `Driver will request a total number of $requestedTotal executor(s).`

This makes more sense:
```
Driver will request a total number of 619 executor(s).
Canceling requests for 4 executor containers
```